pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +696,19 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+			// if the configuration use only one replica we increment
+			if len(cr.Spec.Objects.Servers) != 0 {
```

Comment:
at this point in the code, in this servers loop, we already know len is not 0. are we sure this check is necessary?